app/workers/zipfulldata.rb

```diff
@@ -195,11 +193,11 @@ def create_picture_phenotype_csv(zipfile)
             row << '-'
           end
         end
-        log "Putting a line into CSV"
+        logger.info("Putting a line into CSV")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.